src/services/navigationBar.ts

```diff
@@ -306,19 +309,16 @@ function addNodeWithRecursiveInitializer(node: VariableDeclaration | PropertyAss
     }
 }
 
-/**
- * Historically, we've elided dynamic names from the nav tree (including late bound names),
- * but included certain "well known" symbol names. While we no longer distinguish those well-known
- * symbols from other unique symbols, we do the below to retain those members in the nav tree.
- */
 function hasNavigationBarName(node: Declaration) {
-    return !hasDynamicName(node) ||
-        (
-            node.kind !== SyntaxKind.BinaryExpression &&
-            isPropertyAccessExpression(node.name.expression) &&
-            isIdentifier(node.name.expression.expression) &&
-            idText(node.name.expression.expression) === "Symbol"
-        );
+    const name = getNameOfDeclaration(node);
+    if (name === undefined) return false;
+
+    if (isComputedPropertyName(name)) {
+        const expression = name.expression;
+        return (isPropertyAccessExpression(expression) && isIdentifier(expression.expression) && idText(expression.expression) === "Symbol")
+            || isEntityNameExpression(expression) || isNumericLiteral(expression) || isStringOrNumericLiteralLike(expression);

```

Comment:
```suggestion
        return isEntityNameExpression(expression) || isNumericLiteral(expression) || isStringOrNumericLiteralLike(expression);
```

We no longer need to special case `Symbol.` if we allow any entityname-like expression in the next condition.